### DISCUSSION TITLE
Ethers  ethers@^6.6.4 Support

### DISCUSSION THREAD

**Discussion by u1x30** [NomicFoundation/hardhat#4156] [Troubleshooting]
Hi - can anyone please provide any insight if hardhat-ethers supports ethers@^6.6.4 or when will such support be available?

Getting following error on upgrade:

npm ERR! Found: ethers@6.6.4
npm ERR! node_modules/ethers
npm ERR!   dev ethers@"^6.6.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ethers@"^5.0.0" from @nomiclabs/hardhat-ethers@2.2.3
npm ERR! node_modules/@nomiclabs/hardhat-ethers
npm ERR!   dev @nomiclabs/hardhat-ethers@"^2.2.3" from the root project

Thanks in advance.

**Comment by fvictorio** [comment]
You have to use `@nomicfoundation/hardhat-ethers`, not `@nomiclabs/hardhat-ethers`

**Comment by u1x30** [comment]
Hi fvictorio - greatly appreciate your response; everything is working (including deployments.

Thank you, appreciate your time.

Regards.
